File: TalkBack.Server/TalkBack.Chat.Data/Models/Message.cs
using System.ComponentModel.DataAnnotations;

namespace TalkBack.Chat.Data.Models;

public class Message
{
    public string? SenderId { get; set; }
    public string? ReceiverId { get; set; }
    public string? User { get; set; }
    public string? MsgText { get; set; }

}

File: TalkBack.Client/src/app/models/Message.ts
export class Message {
  senderId: string = '';
  receiverId: string = '';
  user: string = '';
  msgText: string = '';

  assign(senderId?: string, receiverId?: string, user?: string, msgText?: string) {
    this.senderId = senderId == undefined ? '' : senderId;
    this.receiverId = receiverId == undefined ? '' : receiverId;
    this.user = user == undefined ? '' : user;
    this.msgText = msgText == undefined ? '' : msgText;
  }
}